src/gc/handletable.cpp

```diff
@@ -313,6 +313,14 @@ OBJECTHANDLE HndCreateHandle(HHANDLETABLE hTable, uint32_t uType, OBJECTREF obje
     }
 #endif // _DEBUG && !FEATURE_REDHAWK
 
+    // If we are creating a variable-strength handle, verify that the
+    // requested variable handle type is valid.
+    if (uType == HNDTYPE_VARIABLE && !IS_VALID_VHT_VALUE(lExtraInfo))
```

Comment:
The handle table methods are expected to be called with valid arguments. If you call them with invalid arguments, they can succeed, crash, or whether in release builds. It is fine to assert on invalid arguments in debug builds. The error handling in this case should be consistent with the overall design. For example, bad uType passed to HndCreateHandle is not checked in release builds either.